docs/slack-connector/README.md

```diff
@@ -0,0 +1,33 @@
+# Slack Connector Installation<!-- omit in toc -->
+
+- [Overview](#overview)
+- [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+  - [Prerequisites](#prerequisites)
+  - [Steps](#steps)
+
+## Overview
+
+Slack Connector is a component which allows contact from inside of Kyma environment to the Slack API.
+
+## Installation in Kyma with Helm
+
+### Prerequisites
+
+- Connection to Kyma cluster
+- Slack Connector Docker image
+
+### Steps
+
+1. Visit our [authentication page](https://auth-slack.herokuapp.com/). Click `Add to Slack`. You will be redirected to another page. Select your desired workspace and click `Allow`
```

Comment:
```suggestion
1. Go to the [authentication page](https://auth-slack.herokuapp.com/). Click `Add to Slack`. This redirects you to another page. Select your desired workspace and click `Allow`.
```
- Try to avoid using possessive pronouns like "our", "my", etc.
- Don't use future tense (_will_), use present tense.
- Don't use passive voice (_you are redirected_), use active voice (_this redirects you_).
- Punctuate your sentences. Remember to add periods at the end of your sentences. If you use sentences in a list, end them with periods.